### ISSUE TITLE
Configure shorthand payloads?

### ISSUE THREAD

**Issue by xcambar** [samselikoff/ember-cli-mirage#185]
Hi,
I just started using Mirage, and I am wondering how/if one can customize the structure of the payloads when using shorthands.
As an example, by default, `this.get('/contacts')` returns:

```js
{
  contacts: [
    // the list of contacts
  ]
}
```

What I would like to have __by default__ for __all__ my `GET` endpoints can be way more crazy/verbose/simple, such as:

```js
[
    // the list of contacts
]
```

With that use case, all my endpoints look like this (factorisation aside):

```js
  this.get('/contacts', function (db, req) {
    return db.contacts;
  });
```

...which is ok but a bit disappointing as I can't use shorthands :wink: .

__tl;dr;__ Does Mirage provide configurability for the shorthands' payload structure?

Thanks for Mirage, it's very handy.

**Comment by samselikoff** [comment]
Short answer right now is no, bc this really belongs in the serialization layer. We're working on a simple orm so you can have a per-model serializers.

But it'd be possible to add this in now with a simple `includeRoot` option. Several people have asked so I may be able to throw it in before the orm is done - I know it's frustrating!

I'll try to work on this soon :)

**Comment by xcambar** [comment]
Only reasonably frustrating, no big deal.

Though good to know it's planned/coming.

**Comment by samselikoff** [comment]
The [serializers](http://www.ember-cli-mirage.com/docs/v0.2.x/serializers/) are in 0.2.x and solve this problem. 0.2.0 is still in beta but is being worked on. Lmk if you have more questions!

**Comment by xcambar** [comment]
Yes, I've realised that. Thanks for closing the issue.
